kuzzleio/kuzzle
# Update cluster eviction node process

## What does this PR do ?



* This PR fix an issue where when updating a cluster or restarting a node of a cluster would killed all node in the cluster except the new one, instead of just joining the cluster

### How should this be manually tested?


  - Step 1 : before the fix, run `docker compose up -d`
  - Step 2 : `docker compose restart kuzzle_node_2`
  - Step 3 : See that the cluster ends up in a bad state

After the fix, re-run the operation, see that the whole process work w/o any issues

### Other changes



### Boyscout
